src/effects/LensDistortionEffect.js

```diff
@@ -0,0 +1,120 @@
+import { Uniform, Vector2 } from "three";
+import { Effect } from "postprocessing";
+
+import fragmentShader from "./glsl/lens-distortion.frag";
+
+/**
+ * Lens distortion effect.
+ * Original shader ported from https://github.com/ycw/three-lens-distortion
+ * @author danielr electroteque@gmail.com
```

Comment:
```suggestion
```

Author info is already included in the git history and shouldn't be maintained in code.

Reply:
done